### ISSUE TITLE
I feel that the Gradient Fusion and Adapter Fusion approaches are very similar in their thinking

### ISSUE THREAD

**Issue by sunyuhan19981208** [TencentARC/Mix-of-Show#5]
Can I understand Gradient Fusion as LoRA version of Adapter Fusion?

Here is Adapter Fusion paper: https://arxiv.org/pdf/2005.00247.pdf.

**Comment by guyuchao** [comment]
Although I haven't read that paper before, I just came through this paper and found several key differences.

1) As stated in this paper "we **introduce parameters Ψ** that learn to combine theN task adapters to solve the target task." This method should include a new set of parameters, while in our method we just update the pretrained weight to fit all concepts.

2) As stated in this paper "which means that **the training dataset of m is used twice**: once for training the adapters Φm and again for **training Fusion parameters Ψm**, which learn to compose the information stored in the N task adapters." However, in our setting, the training sets are not available. We cannot collect the concept data from each users, therefore, our optimization perform layer-wise updatation without accessing original data, which is the essence of "decentralize".

For the high-level idea, we want to collect several independent ED-LoRAs to jointly support those concepts **without accessing original data** and keeping each ED-LoRAs' quality. This two-step pipeline shares some similarities with Adapter Fusion, but it is primarily derived from federated/decentralized learning.
